mathics_scanner/data/named-characters.yml

```diff
@@ -4469,15 +4469,15 @@ Infinity:
   wl-unicode-name: INFINITY
 
 # Infix isn't really an operator
-# See also Tilde
+# See also RawTilde
 Infix:
   ascii: "~"
   amslatex: "\\textasciitilde"
   has-unicode-inverse: false
   is-letter-like: false
 # operator-name: Infix
   unicode-equivalent: "~"
-  wl-unicode-name: TILDE
+#  wl-unicode-name: RAWTILDE
```

Comment:
This comment is confusing. The one above says "See also RawTilde" so this isn't that. What purpose does the comment serve here?